### PR TITLE
sql/tidb-specific: update ddl reorg parameter

### PR DESCRIPTION
**Pull request by crazycs520**
update `tidb_ddl_reorg_worker_cnt` and `tidb_ddl_reorg_batch_size` parameter definition。

### CODE REVIEW

**Comment by crazycs520** [comment]
@winkyao @zimulala PTAL

File: sql/tidb-specific.md [hunk 1]
```diff
@@ -339,12 +339,20 @@ set @@global.tidb_distsql_scan_concurrency = 10
...
 
 这个变量用来设置 DDL 操作 re-organize 阶段的并发度。
 
+### tidb_ddl_reorg_batch_size
+
+作用域: GLOBAL
+
+默认值：1024
+
+这个变量用来设置 DDL 操作 re-organize 阶段的 batch size 大小。
```
Comment (winkyao) [inline sql/tidb-specific.md]: 添加说明，增大跟缩小的一些原理跟提示

File: v2.1/sql/tidb-specific.md [hunk 2]
```diff
@@ -339,12 +339,20 @@ set @@global.tidb_distsql_scan_concurrency = 10
...
 
 这个变量用来设置 DDL 操作 re-organize 阶段的并发度。
 
+### tidb_ddl_reorg_batch_size
+
+作用域: GLOBAL
+
+默认值：1024
+
+这个变量用来设置 DDL 操作 re-organize 阶段的 batch size 大小。
```
Comment (winkyao) [inline v2.1/sql/tidb-specific.md]: ditto

**Comment by crazycs520** [comment]
@winkyao PTAL again

**Comment by lilin90** [comment]
@CaitinChen Please help review this PR and make the description clearer. Thanks!

File: sql/tidb-specific.md [hunk 3]
```diff
@@ -339,12 +339,20 @@ set @@global.tidb_distsql_scan_concurrency = 10
...
 
 这个变量用来设置 DDL 操作 re-organize 阶段的并发度。
 
+### tidb_ddl_reorg_batch_size
+
+作用域: GLOBAL
+
+默认值：1024
+
+这个变量用来设置 DDL 操作 re-organize 阶段的 batch size 大小。比如 Add Index 操作，需要回填索引数据，通过并发 tidb_ddl_reorg_worker_cnt 个 worker 一起回填数据，每个 worker 是一个 batch 一个 batch 的回填的。如果 add index 时有 update 操作或者 replace 等更新操作较多，batch size 越大，事务冲突的概率也会越大，此时建议改小 batch size 的大小，最小值是32。在没有事务冲突的情况下，batch size 可以设置大些，最大值是10240，这样回填数据的速度更快，当然 tikv 的写入压力也会变大。
```
Comment (CaitinChen) [inline sql/tidb-specific.md]:
```suggestion
这个变量用来设置 DDL 操作 re-organize 阶段的 batch size。比如 Add Index 操作，需要回填索引数据，通过并发 tidb_ddl_reorg_worker_cnt 个 worker 一起回填数据，每个 worker 以 batch 为单位进行回填。如果 Add Index 时有较多 Update 操作或者 Replace 等更新操作，batch size 越大，事务冲突的概率也会越大，此时建议调小 batch size 的值，最小值是 32。在没有事务冲突的情况下，batch size 可设为较大值，最大值是 10240，这样回填数据的速度更快，但是 TiKV 的写入压力也会变大。
```
```suggestion
这个变量用来设置 DDL 操作 re-organize 阶段的 batch size 大小。比如 Add Index 操作，需要回填索引数据，通过并发 tidb_ddl_reorg_worker_cnt 个 worker 一起回填数据，每个 worker 是一个 batch 一个 batch 的回填的。如果 add index 时有 update 操作或者 replace 等更新操作较多，batch size 越大，事务冲突的概率也会越大，此时建议改小 batch size 的大小，最小值是32。在没有事务冲突的情况下，batch size 可以设置大些，最大值是10240，这样回填数据的速度更快，当然 tikv 的写入压力也会变大。
```

Comment (CaitinChen) [reply sql/tidb-specific.md]: “一个 batch 一个 batch” sounds informal.
"batch size" is "batch 的大小", which means "batch size 的值".

File: v2.1/sql/tidb-specific.md [hunk 4]
```diff
@@ -339,12 +339,20 @@ set @@global.tidb_distsql_scan_concurrency = 10
...
 
 这个变量用来设置 DDL 操作 re-organize 阶段的并发度。
 
+### tidb_ddl_reorg_batch_size
+
+作用域: GLOBAL
+
+默认值：1024
+
+这个变量用来设置 DDL 操作 re-organize 阶段的 batch size 大小。比如 Add Index 操作，需要回填索引数据，通过并发 tidb_ddl_reorg_worker_cnt 个 worker 一起回填数据，每个 worker 是一个 batch 一个 batch 的回填的。如果 add index 时有 update 操作或者 replace 等更新操作较多，batch size 越大，事务冲突的概率也会越大，此时建议改小 batch size 的大小，最小值是32。在没有事务冲突的情况下，batch size 可以设置大些，最大值是10240，这样回填数据的速度更快，当然 tikv 的写入压力也会变大。
```
Comment (CaitinChen) [inline v2.1/sql/tidb-specific.md]:
```suggestion
这个变量用来设置 DDL 操作 re-organize 阶段的 batch size。比如 Add Index 操作，需要回填索引数据，通过并发 tidb_ddl_reorg_worker_cnt 个 worker 一起回填数据，每个 worker 是以 batch 为单位进行回填。如果 Add Index 时有较多 Update 操作或者 Replace 等更新操作，batch size 越大，事务冲突的概率也会越大，此时建议调小 batch size 的值，最小值是 32。在没有事务冲突的情况下，batch size 可设为较大值，最大值是 10240，这样回填数据的速度更快，但是 TiKV 的写入压力也会变大。
```

**Comment by CaitinChen** [comment]
@crazycs520 PTAL

**Pull request merged**